nautobot/extras/graphql/types.py

```diff
@@ -1,9 +1,31 @@
+import graphene
+from graphene_django import DjangoObjectType
 import graphene_django_optimizer as gql_optimizer
 
 from nautobot.extras.models import Status, Tag, DynamicGroup
 from nautobot.extras.filters import StatusFilterSet, TagFilterSet, DynamicGroupFilterSet
 
 
+class NautobotObjectType(DjangoObjectType):
+    url = graphene.String()
+
+    def resolve_url(self, args):
+        return self.get_absolute_url(api=True)
```

Comment:
Probably would be good to add some error handling here in case it somehow gets a model class that doesn't implement `get_absolute_url(api=True)`?

Reply:
Looks like this was maybe originally based on next and then moved to develop. We don't have the api kwarg in `get_absolute_url` on develop.

Reply:
May need to rework this then, as the *goal* of the issue, AIUI, was to return the REST API URL for the object - the UI URL is less useful in a GraphQL context. Maybe use `get_route_for_model` in 1.x?